csrc/scheduler/mma_utils.cpp

```diff
@@ -379,11 +406,6 @@ bool canValidateIsInnerDim(
       if (!split->factor()->isConstInt()) {
         return false;
       }
-      if (split->factor()->evaluateInt() < inner_dim_size) {
```

Comment:
This check had to be removed? This looks like a part of inner-CTA reduction functionality.

Reply:
This check leads to a false positive error due to the dryrun swizzle of mma output tensor.
```
scheduleTuringOperandRead tv= T3_l[ iS37{( ceilDiv(i0, 128) )}, bS39{( ceilDiv(1, 128) )}, iS41{( ceilDiv(i2, 32) )}, iS113{( ceilDiv(32, 16) )}, iS118{( ceilDiv(( ( ceilDiv(128, 8) ) * 8 ), 64) )}, bS126{( ceilDiv(( ( ( ceilDiv(( ceilDiv(128, 8) ), 8) ) * 8 ) * 8 ), 64) )}, iS128{( ceilDiv(64, 16) )}, bS130{( ceilDiv(64, 8) )}, bS131{8}, iS129{16}, iS114{16} ]
Split factor too small Split: iS38{128} by factor 8 -> iS115{( ceilDiv(128, 8) )}, iS116{8}, start offset: 0, stop offset: 0

unknown file: Failure
C++ exception with description "canValidateIsInnerDim(m_dims.back(), tv->axis(-2), 16) INTERNAL ASSERT FAILED at "/opt/pytorch/nvfuser/csrc/scheduler/mma_utils.cpp":886, please report a bug to PyTorch. MMA swizzle: requires instruction tile iterdomains on the innermost side of the tensordomain
```